test/end2end_test.go

```diff
@@ -888,6 +888,23 @@ type lazyConn struct {
 	beLazy int32
 }
 
+// errMsgContainsPossibleStopMessages addresses around the timing of when TCP packets
+// are sent from client to server, and when we tell the server to stop, so we need to account for both
+// of these possible error messages:
+//  1. If the call to ss.S.Stop() causes the server's sockets to close while there's still in-fight
+//     data from the client on the TCP connection, then the kernel can send an RST back to the client (also
+//     see https://stackoverflow.com/questions/33053507/econnreset-in-send-linux-c). Note that while this
+//     condition is expected to be rare due to the rpcStartedOnServer synchronization, in theory it should
+//     be possible, e.g. if the client sends a BDP ping at the right time.
+//  2. If, for example, the call to ss.S.Stop() happens after the RPC headers have been received at the
+//     server, then the TCP connection can shutdown gracefully when the server's socket closes. raceyness around checks if the error message when checking reading frames
+const possibleConnResetMsg = "connection reset by peer"
```

Comment:
these variables are used for error messages below. I tried  returning the error message as is from the func like ok, errMsg := errMsgContainsPossibleStopMessages.. But the other usage in `TestDetailedConnectionCloseErrorPropagatesToRpcError` starts to look ugly.. Let me know if you can think about a better way

Reply:
No this seems fine to me.